cpp/CMakeLists.txt

```diff
@@ -124,6 +124,9 @@ else()
   message(STATUS "Manually setting BLAS to ${BLAS_LIBRARIES}")
 endif()
 
+set(Protobuf_USE_STATIC_LIBS ON)
+find_package(Protobuf REQUIRED)
+
```

Comment:
This is not going to work if you don't add protobuf to the CI scripts and the conda recipe